saleor/checkout/utils.py

```diff
@@ -175,18 +184,29 @@ def add_variants_to_checkout(
     country_code = checkout.get_country()
 
     checkout_lines = checkout.lines.select_related("variant")
-    variant_ids_in_lines = {line.variant_id: line for line in checkout_lines}
+
+    lines_by_variant_id = {line.variant_id: line for line in checkout_lines}
```

Comment:
I see naming incosistency here. You have `lines_by_variant_id` here and `lines_by_ids` - `id` vs `ids`.